Detectors/TOF/simulation/include/TOFSimulation/Digitizer.h

```diff
@@ -77,18 +85,41 @@ class Digitizer
   Float_t mEffBoundary3;
 
   // info TOF timewindow
-  Int_t mTimeFrameCurrent;
+  Int_t mReadoutWindowCurrent;
   Double_t mEventTime;
   Int_t mEventID = 0;
   Int_t mSrcID = 0;
 
+  bool mContinuous = false;
+
   // digit info
-  std::vector<Digit>* mDigits;
-  o2::dataformats::MCTruthContainer<o2::tof::MCLabel>* mMCTruthContainer =
-    nullptr; ///< Array for MCTruth information associated to digits in mDigitsArrray. Passed from the digitization
+  //std::vector<Digit>* mDigits;
+
+  static const int MAXWINDOWS = 10; // how many readout windows we can buffer
```

Comment:
This is a preliminary number. It will be tuned later on. For now we will test 10, but we will probably reduce it according to the outcome of the simulations.